### ISSUE TITLE
Demo on pybbm.org is broken

### ISSUE THREAD

**Issue by matthijskooijman** [hovel/pybbm#287]
It seems the domain has expired and is hijacked, now showing ad links? http://www.pybbm.org/

**Comment by lampslave** [comment]
Sad but true. We don't have access to this domain anymore :(

**Comment by matthijskooijman** [comment]
Maybe the link should be removed? Or a demo hosted at some other place? Now it makes it look like this project is no longer maintained...

**Comment by lampslave** [comment]
Yes, I'll remove it. We still use this project in our services, so it's maintained although it isn't under heavy development.

**Comment by lampslave** [comment]
Check https://archlinux.org.ru/ (https://github.com/hovel/aor) if you want to see live demo.

**Comment by DylannCordel** [comment]
:+1: here : We still use pybbm too. Each time we have a new project with a builtin-board (not often unfortunately), we give time to enhance this app.
